xml/operations-maintenance-recover-boot-processes.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0"?>
+<?xml-stylesheet href="urn:x-suse:xslt:profiling:docbook51-profile.xsl"
+ type="text/xml"
+ title="Profiling step"?>
+<!DOCTYPE section [
+ <!ENTITY % entities SYSTEM "entity-decl.ent"> %entities;
+]>
+<section xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="recover_failed_boot_processes">
+ <title>Recover Start-up Processes</title>
+ <para>
+  In this scenario, processes do not start. If those processes are not running,
```

Comment:
processes do not start? Like, automatically?

Reply:
In some cases services are not configured to autostart on node (re)boot to avoid scenarios that could lead to data loss or corruption without necessary manual intervention.